src/Evolution/Executables/GrMhd/GhValenciaDivClean/EvolveGhValenciaDivClean.hpp

```diff
@@ -0,0 +1,92 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <vector>
+
+#include "ApparentHorizons/Tags.hpp"
+#include "Domain/Creators/RegisterDerivedWithCharm.hpp"
+#include "Domain/Creators/TimeDependence/RegisterDerivedWithCharm.hpp"
+#include "Domain/FunctionsOfTime/RegisterDerivedWithCharm.hpp"
+#include "Evolution/DiscontinuousGalerkin/Limiters/Tags.hpp"
+#include "Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/RegisterDerivedWithCharm.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/Tags.hpp"
+#include "Evolution/Systems/GrMhd/GhValenciaDivClean/BoundaryConditions/RegisterDerived.hpp"
+#include "Evolution/Systems/GrMhd/GhValenciaDivClean/BoundaryCorrections/RegisterDerived.hpp"
+#include "Evolution/Systems/GrMhd/ValenciaDivClean/Tags.hpp"
+#include "Evolution/VariableFixing/Tags.hpp"
+#include "NumericalAlgorithms/Interpolation/Callbacks/ObserveTimeSeriesOnSurface.hpp"
+#include "NumericalAlgorithms/Interpolation/InterpolationTarget.hpp"
+#include "NumericalAlgorithms/Interpolation/Tags.hpp"
+#include "Parallel/RegisterDerivedClassesWithCharm.hpp"
+#include "Utilities/TMPL.hpp"
+
+template <typename InitialData, typename... InterpolationTargetTags>
+struct EvolutionMetavars
+    : public virtual GhValenciaDivCleanDefaults,
```

Comment:
I want all of the type aliases in `GhValenciaDivCleanDefaults` to be brought into the `EvolutionMetavars` directly, without having to specify `using ....` statements for each type alias. The type aliases are brought in automatically for public inheritance from the non-dependent `GhValenciaDivCleanDefaults`, but not from the template `GhValenciaDivCleanTemplateBase` that depends on the `EvolutionMetavars` template parameters -- this is ultimately why I have the two separate base classes.

Then, the virtual inheritance is because both the `EvolutionMetavars` and `GhValenciaDivCleanTemplateBase` inherit from `GhValenciaDivCleanDefaults` (the type aliases therein are also used in some definitions in `GhValenciaDivCleanTemplateBase`), so needs `virtual` to avoid inheritance ambiguities.